docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/supabase/page.mdx

---
title: 'Supabase'
description: 'Stream data from Supabase into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# Supabase

Supabase is an open-source backend platform built on PostgreSQL, offering APIs, authentication, and real-time capabilities. It provides a Firebase-like experience with SQL-based data storage.

## Using Supabase with GlassFlow

GlassFlow can ingest changes from Supabase into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, joins, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** Supabase connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** Supabase connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
